tools/integration_tests/operations/operations_test.go

```diff
@@ -49,14 +49,16 @@ func TestMain(m *testing.M) {
 	setup.ExitWithFailureIfBothTestBucketAndMountedDirectoryFlagsAreNotSet()
 
 	if setup.TestBucket() != "" && setup.MountedDirectory() != "" {
-		log.Printf("Both --testbucket and --mountedDirectory can't be specified at the same time.")
+		log.Print("Both --testbucket and --mountedDirectory can't be specified at the same time.")
 		os.Exit(1)
 	}
 
 	// Run tests for mountedDirectory only if --mountedDirectory flag is set.
 	setup.RunTestsForMountedDirectoryFlag(m)
 
 	// Run tests for testBucket
+	setup.SetUpTestDirForTestBucketFlag()
```

Comment:
what is this change for??

Reply:
Before...we were building gcsfuse again for every mounting e.g. static and only dir...

Put it in beginning place only ...so that now gcsfuse is building only once for every test package
Before gcsfuse was building twice for every test package...

I made it one